packages/mui-base/src/Portal/Portal.test.tsx

```diff
@@ -80,24 +80,24 @@ describe('<Portal />', () => {
       </div>,
     );
     const rootElement = document.querySelector('#test1');
```

Comment:
You should be able to do `const rootElement = document.querySelector<HTMLDivElement>('#test1');` instead of casting.